osvr/RenderKit/RenderManagerD3DBase.cpp

```diff
@@ -897,17 +897,17 @@ namespace renderkit {
 
                 v.TexR.x = meshVertex.m_texRed[0];
                     v.TexR.y =
-                        RenderManager::DistortionMeshVertex::flipTexCoord(
+                        osvr::renderkit::DistortionMeshVertex::flipTexCoord(
```

Comment:
Since we're already inside the `osvr::renderkit` namespace, you could omit it entirely and just write `DistortionMeshVertex::flipTexCoord(...);`.